src/ast/field_analyser.cpp

```diff
@@ -40,6 +40,15 @@ void FieldAnalyser::visit(Jump &jump __attribute__((unused)))
 {
 }
 
+void FieldAnalyser::check_kfunc_args(void)
+{
+  if (has_kfunc_probe_ && has_mixed_args_)
+  {
+    error("Failed: probe has attach points with mixed arguments",
```

Comment:
Having `Failed: ` seems unnecessary since there should already be a big `ERROR: ` printed. My 2c

Reply:
ok, will change, thanks